paasta_tools/cli/cmds/validate.py

```diff
@@ -108,6 +109,37 @@ def get_schema(file_type):
     return json.loads(schema)
 
 
+def validate_instance_names(config_file_object, file_path):
+    return_code = True
+    for instance_name in config_file_object:
+        if (
+            not instance_name.startswith("_")
+            and len(sanitise_kubernetes_name(instance_name)) > 63
+        ):
+            return_code = False
+            paasta_print(
```

Comment:
I think you want to have a list to record the instances violating the restriction together, so that you can just `paasta_print` the error message once. Also, by doing that, you can do `return len(list) == 0`.